cts/cts-fencing.in

```diff
@@ -284,6 +284,11 @@ class Test(object):
         if args['kill']:
             if self.verbose:
                 print("Also running: "+args['kill'])
+            ### Typically, the kill argument is used to detect some sort of
+            ### failure. Without yielding for a few seconds here, the process
+            ### launched earlier that is listening for the failure may not have
+            ### time to connect to pacemaker-execd.
+            time.sleep(2)
```

Comment:
I worry this will make the test take forever. Can we do this conditionally if something fails?

Reply:
Right, this is just code I'm moving around.  It dates back to 2012 (!), added as part of fd2c81800d to fix cl#5094.  This should only happen if add_cmd_and_kill is used, which happens five times.  So it doesn't seem like it should add much delay.  On the other hand,  it's really old code so it'd be worth removing the delay and seeing what happens.